R/importance_perm.R

```diff
@@ -43,6 +43,37 @@
 #'
 #' For censored data, importance is computed for each evaluation time (when a
 #' dynamic metric is specified).
+#'
+#' By default, no parallelism is used to process models in \pkg{tune}; you have
+#' to opt-in.
+#'
+#' ## Using future to parallel process
+#'
+#' You should install the package and choose your flavor of parallelism using
+#' the [plan][future::plan] function. This allows you to specify the number of
+#' worker processes and the specific technology to use.
+#'
+#' For example, you can use:
+#'
+#' ```r
+#'    library(future)
+#'    plan(multisession, workers = 4)
+#' ```
+#' and work will be conducted simultaneously (unless there is an exception; see
+#' the section below).
+#'
+#' See [future::plan()] for possible options other than `multisession`.
+#'
+#' ## Using mirai  to parallel process
+#'
+#' To set the specific for parallel processing with \pkg{mirai}, use the
```

Comment:
```suggestion
#' To configure parallel processing with \pkg{mirai}, use the
```

Or clarify "specific ____"